.github/workflows/publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and Publish
+
+on:
+    push:
+      tags:
+        - '*'
```

Comment:
On suggestion i would like to provide here is that you can just disable the step which runs publish command. Instead of disabling the entire workflow. You can look at form-controls as an example

Reply:
Hey Mohan, could you explain a bit more. I couldn't follow